spec/Section 6 -- Execution.md

```diff
@@ -322,6 +366,49 @@ Unsubscribe(responseStream):
 
 - Cancel {responseStream}
 
+## Yield Subsequent Payloads
+
+If an operation contains subsequent payload records resulting from `@stream` or
+`@defer` directives, the {YieldSubsequentPayloads} algorithm defines how the
+payloads should be processed.
+
+YieldSubsequentPayloads(initialResponse, subsequentPayloads):
+
+- Let {initialRecords} be any items in {subsequentPayloads} with a completed
+  {dataExecution}.
+- Initialize {initialIncremental} to an empty list.
+- For each {record} in {initialRecords}:
+  - Remove {record} from {subsequentPayloads}.
+  - If {isCompletedIterator} on {record} is {true}:
+    - Continue to the next record in {records}.
+  - Let {payload} be the completed result returned by {dataExecution}.
+  - Append {payload} to {initialIncremental}.
+- If {initialIncremental} is not empty:
+  - Add an entry to {initialResponse} named `incremental` containing the value
+    {incremental}.
+- Yield {initialResponse}.
+- While {subsequentPayloads} is not empty:
+- If a termination signal is received:
+  - For each {record} in {subsequentPayloads}:
+    - If {record} contains {iterator}:
+      - Send a termination signal to {iterator}.
+  - Return.
+- Wait for at least one record in {subsequentPayloads} to have a completed
+  {dataExecution}.
+- Let {subsequentResponse} be an unordered map with an entry {incremental}
+  initialized to an empty list.
+- Let {records} be the items in {subsequentPayloads} with a completed
+  {dataExecution}.
+  - For each {record} in {records}:
+    - Remove {record} from {subsequentPayloads}.
+    - If {isCompletedIterator} on {record} is {true}:
+      - Continue to the next record in {records}.
+    - Let {payload} be the completed result returned by {dataExecution}.
+    - Append {payload} to {incremental}.
```

Comment:
This should say the `{incremental}` entry on `{subsequentReponse}` or something?

Reply:
updated